### ISSUE TITLE
PassthroughSubject created without combine annotation

### ISSUE THREAD

**Issue by sidepelican** [uber/mockolo#184]
This protocol has no `combine` annotation, but generated mock has special implementation using `PassthroughSubject` .

- source 
```swift
/// @mockable
protocol Foo {
    var bar: AnyPublisher<Int, Never> { get }
}
```

- generated
```swift
class FooMock: Foo {
    init() { }


    var bar: AnyPublisher<Int, Never> { return self.barSubject.eraseToAnyPublisher() }
    private(set) var barSubject = PassthroughSubject<Int, Never>()
}
```

I want to set my own `AnyPublisher` to mock object.
It seems that there are no way to disable generating special mock for Combine.

mockolo version: 1.7.0

**Comment by maxwellE** [comment]
I created this PR https://github.com/uber/mockolo/pull/186

**Comment by maxwellE** [comment]
Merged #186

**Comment by sidepelican** [comment]
Thank you!!
